site/docs/configuration.md

```diff
@@ -43,6 +43,7 @@ Iceberg tables support table properties to configure table behavior, like the de
 | write.metadata.metrics.default     | truncate(16)       | Default metrics mode for all columns in the table; none, counts, truncate(length), or full |
 | write.metadata.metrics.column.col1 | (not set)          | Metrics mode for column 'col1' to allow per-column tuning; none, counts, truncate(length), or full |
 | write.target-file-size-bytes       | Long.MAX_VALUE     | Controls the size of files generated to target about this many bytes. |
+| write.disable-nullability-check    | false              | Disables the nullable check on fields              |
```

Comment:
I thought this table configuration would be removed? It doesn't make sense to me that you would never want to check nullability for a table. If Spark has a problem, then it should be turned off for individual Spark jobs, or for all Spark jobs in a Spark config.

Reply:
Also, this is still phrased as a negative, "disable", which we should avoid.

Reply:
It is removed. I just forgot to modify this section. 👍